draft-ietf-rats-architecture.md

```diff
@@ -430,10 +431,9 @@ is the kernel to be booted.  The final Evidence thus contains two sets of
 Claims: one set about the bootloader as measured and signed by the BIOS,
 plus a set of Claims about the kernel as measured and signed by the bootloader.
 
-This example could be extended further
-by, say, making the kernel become another Attesting Environment for
-an application as another Target Environment, resulting in a third set
-of Claims in the Evidence pertaining to that application.
+This example could be extended further by making the kernel become another
+Attesting Environment for an application as another Target Environment. This
+would result in a third set of Claims in the Evidence pertaining to that application.
```

Comment:
```suggestion
This example could be extended further by making the kernel become another
Attesting Environment for an application as another Target Environment. 
This would result in a third set of Claims in the Evidence pertaining to that application.
```